.gitattributes

```diff
@@ -1,2 +1,16 @@
 website export-ignore
 examples export-ignore
+
+# We will not export unreleased components.
```

Comment:
Please also exclude `integrations` and `bin`.

Reply:
Updated.